test/testUtils/mayaUtils.py

```diff
@@ -167,6 +167,9 @@ def openGroupBallsScene():
 def openPrimitivesScene():
     return openTestScene("reorderCmd", "primitives.ma" )
 
+def openCompositionArcsScene():
+    return openTestScene("compositionArcs", "compitionArcs.ma" )
```

Comment:
Forgot to change here.

Reply:
Addressed in 88d344606b273390a0ecccea1db2a736949e758b